### ISSUE TITLE
SwipeCards.js Unexpected token (164:6)

### ISSUE THREAD

**Issue by budda** [meteor-factory/react-native-tinder-swipe-cards#42]
npm version results in a compile failure.

```
Error in ./~/react-native-swipe-cards/SwipeCards.js
Module parse failed: /Users/mike/Code/tinderreact/node_modules/react-native-swipe-cards/SwipeCards.js Unexpected token (164:6)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (164:6)
 @ ./src/tinder.js 18:29-64
```

**Comment by MadeInMoon** [comment]
I guess this error points the jsx expression `<Defaults.NoMoreCards />`at line 164, so it should be a config problem (jsx loader?).
How did you initiate your project?

At least try a create a new project by cli with `react-native init`, and look the **.babelrc**, it should contain 
`{
"presets": ["react-native"]
}`

**Comment by vitorebatista** [comment]
@budda could you test with the last merged PR #47 ?
